en/tools/config-catalog-generator/dist/config-catalog.md

```diff
@@ -2307,7 +2307,7 @@ enable_token_hashing = false</code></pre>
                                         
                                     </div>
                                     <div class="param-description">
-                                        <p>List of allowlisted scopes. Take desc from Key Concepts page.</p>
+                                        <p>List of allowed scopes. Take desc from Key Concepts page.</p>
```

Comment:
@RakhithaRR , please check this change. I feel like allowedlisted is correct because whitelisted was changed to allowed listed recently.